controllers/argocd/ingress.go

```diff
@@ -246,75 +246,19 @@ func (r *ReconcileArgoCD) reconcileGrafanaIngress(cr *argoproj.ArgoCD) error {
 	if argoutil.IsObjectFound(r.Client, cr.Namespace, ingress.Name, ingress) {
 		if !cr.Spec.Grafana.Enabled || !cr.Spec.Grafana.Ingress.Enabled {
 			// Ingress exists but enabled flag has been set to false, delete the Ingress
-			return r.Client.Delete(context.TODO(), ingress)
+			return nil
```

Comment:
Correct me if I'm wrong, but I believe this `r.client.Delete` call still exist: even though we have deprecated Grafana, we should still clean up any old Ingresses that people might have had in the past.

Reply:
Yes @jgwest as e2e test for failing, I removed it (as e2e tests were flaky before, I didn't know the reason), added them back